packages/core/core/src/RequestTracker.js

```diff
@@ -886,6 +956,40 @@ export class RequestGraph extends ContentGraph<
         // to requests as invalidations for future requests.
         this.removeNode(nodeId);
       }
+
+      let packageKeyNodes = this.packageKeyNodes.get(_filePath);
+      if (packageKeyNodes && (type === 'delete' || type === 'update')) {
+          for (let nodeId of packageKeyNodes) {
+          let isInvalid = type === 'delete';
```

Comment:
/nit this indentation is off - should prettier be run against this change?

Reply:
Is it? Prettier is passing for me and I have it setup. Not sure what you're referring to?